sagemaker/launch/launch_partition.py

```diff
@@ -1,11 +1,14 @@
 """ Launch SageMaker training task
 """
 import logging
+import os
 from time import strftime, gmtime
 
 import boto3 # pylint: disable=import-error
 from sagemaker.processing import ScriptProcessor
-import sagemaker
+from sagemaker.workflow.pipeline import Pipeline, PipelineDefinitionConfig
+from sagemaker.workflow.pipeline_context import PipelineSession
+from sagemaker.workflow.steps import ProcessingStep, ProcessingInput
```

Comment:
A quick question about the session here: since I see in the overall description we are moving our partition pipeline to the sagemaker pipeline. Will it affect our previous usage? Like if we can still go the SageMaker processing job console to check the job status. Will it affect any user experience?

Reply:
I see all the previous ScriptProcessor parts are removed.

Reply:
I added this as a demonstration, will probably remove. from the pr description:

>One note about the PR: The changes to the partition launch that use a SageMaker Pipeline are for demonstration purposes, I think I'll remove them alltogether and just have separate partition/gbconvert jobs. But we might want to have an example of how to programmatically build an SM pipeline as an example, e.g. from gsprocessing to training (as SM jobs)

Reply:
If this is just for demonstration purpose, I would suggest we do not directly merge it into main branch. We can create a dev branch first.

Reply:
The code here can be a good example of using SageMaker Pipeline to automate the SageMaker tasks.

Reply:
Yes, I'll create another revision of this PR that does that, and leave this change in a dev branch on my fork. In the future we can add a separate launch script that creates an end2end SageMaker pipeline.